Using TensorFlow backend.
Seed value is 46
MaxAbs

Traceback (most recent call last):
  File "/home/rice/jmc32/DNN_for_Pt_Assignment-master/DNN_Hyperparameters/TestandTrain1NetConfig1AI.py", line 35, in <module>
    import macros_AWS
  File "/home/rice/jmc32/DNN_for_Pt_Assignment-master/DNN_Hyperparameters/macros_AWS.py", line 55, in <module>
    from TestandTrain1NetConfig1AI.py import x_train_prescale, y_train,x_test_prescale,y_test
  File "/home/rice/jmc32/DNN_for_Pt_Assignment-master/DNN_Hyperparameters/TestandTrain1NetConfig1AI.py", line 113, in <module>
    model = KerasClassifier(build_fn=create_model, nb_epoch=5, batch_size=100, verbose=0)
NameError: name 'create_model' is not defined
('True negative total is ', 40837, 'out of', 100000)
('True positive total is ', 52471)
('False negative total is ', 3791)
('False positive total is ', 2901)
('Any missed ones? ', 0)
